webmin/index.cgi

```diff
@@ -61,6 +61,13 @@ push(@wlinks, "edit_ssl.cgi", "edit_ca.cgi");
 push(@wtitles, $text{'ssl_title'}, $text{'ca_title'});
 push(@wicons, "images/ssl.gif", "images/ca.gif");
 
+# Enfoce to disable Webmin updates
+if ($gconfig{'noselfwebminup'}) {
```

Comment:
Instead of changing %access, better to change the %disallow hash below

Reply:
Fixed too.